.circleci/config.yml

```diff
@@ -525,35 +525,21 @@ jobs:
       - checkout:
           path: /home/circleci/project/
       - setup_remote_docker
+      - restore_cache:
+          key: flowetl-deps-1-{{ checksum "flowetl/Pipfile.lock"}}
+      - run:
+          name: Install pipenv
+          command: pip install --upgrade pip pipenv
       - run:
-          name: test running with specified UID/GID for flowetl
+          name: Install python dependencies
           command: |
-            expecteduser=1003
-            expectedgroup=1002
-            gotuser=`docker run --user ${expecteduser}:${expectedgroup} flowminder/flowetl:${CIRCLE_SHA1} bash -c 'id -u'`
-            gotgroup=`docker run --user ${expecteduser}:${expectedgroup} flowminder/flowetl:${CIRCLE_SHA1} bash -c 'id -g'`
-            if [ $gotuser -ne $expecteduser -a  $gotgroup -ne $expectedgroup ]; then
-              echo "Error: not running as correct user and group."
-              exit 1
-            else
-              echo "Running as correct user and group."
-            fi
-      - run: 
-          name: test that airflow picks up dag
+            PIPENV_PIPFILE=flowetl/Pipfile pipenv install --deploy --dev
+      - run:
+          name: run flowetl tests
           command: |
-            # temporary test to make sure dags getting picked up by airflow
-            # this will be tested via a pytest integration test eventually 
-            docker run --name flowetl -d flowminder/flowetl:$CIRCLE_SHA1
-            sleep 5
-            expecteddag=sampledag
-            gotdag=`docker exec -it flowetl airflow list_dags | tail -2 | head -1 | tr -d '[:space:]'`
-            if [ $gotdag != $expecteddag ]; then
-              echo "Error: expected DAG not present."
-              exit 1
-            else
-              echo "Expected DAG present."
-            fi
-            docker stop flowetl
+            cd flowetl && FLOWETL_TAG=$CIRCLE_SHA1 AIRFLOW_HOME=./test_airflow_home pipenv run pytest --junit-xml=test_results/pytest/results.xml ./tests
```

Comment:
Don't need the `&&` because of the `|` above. Could also stick the env vars into the environment for the command?

Reply:
@greenap What do you mean by "don't need the `&&`"? Put the two commands onto separate lines?

Reply:
```
    environment:
      FLOWETL_TAG: $CIRCLE_SHA1 
      AIRFLOW_HOME: ./test_airflow_home 
```

Can I access the circle env var here?

Reply:
Yup.

Reply:
Doesn't seem like you can reference `$CIRCLE_SHA1`...